### PR TITLE
Fixup ABI checker

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: .github/actions/abi-compat/entrypoint.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ cmake -DENABLE_DEVELOPER_DEFAULTS=On -DCMAKE_CXX_FLAGS="-Og" -DCMAKE_BUILD_TYPE=
 make -j 3 umpire
 cd ..
 
-git checkout origin/main
```
Comment (mcfadden8) [inline .github/actions/abi-compat/entrypoint.sh:10]: So what tid this wind up doing then?

Comment (davidbeckingsale) [reply .github/actions/abi-compat/entrypoint.sh:10]: Nothing, for some reason...

File: .github/actions/abi-compat/entrypoint.sh [hunk 2]
```diff
@@ -18,4 +19,4 @@ cd ..
 abi-dumper build-main/lib/libumpire.so -o umpire-main.dump -lver 3.0.0
 abi-dumper build/lib/libumpire.so -o umpire-dev.dump -lver 4.0.0
 
-abi-compliance-checker -l umpire -old umpire-main.dump -new umpire-dev.dump
+abi-compliance-checker -l umpire -old umpire-main.dump -new umpire-dev.dump || true
```
Comment (mcfadden8) [inline .github/actions/abi-compat/entrypoint.sh:22]: Where we will see the failure indication if the abi-compliance-checker fails?

Comment (davidbeckingsale) [reply .github/actions/abi-compat/entrypoint.sh:22]: Good point. Let me consider how to do this.

**Pull request closed without merging**